### DISCUSSION TITLE
Default highlight group for mini.cursorword

### DISCUSSION THREAD

**Discussion by GnikDroy** [nvim-mini/mini.nvim#418] [Ideas]
What do you think about using the `hl-CursorLine` as the default highlight group? This gives a much better contrast than `{ underline = true }` and hopefully works more universally. CursorLine is also a more intuitive highlight group for CursorWord, since the same highlight is applied to a single word instead of a line :)

The one disadvantage is if the user has enabled `cursorline`, words won't look highlighted on the same line. But similar issues are preset with `{ underline = true }` as well. `vim.diagnostic.config()`, for instance, has an underline option. So, a conflict in certain situations is unavoidable.

**Comment by echasnovski** [comment]
Hmm... I am going to go with "decisively disagree". For three reasons:

- Changing highlight group is one of the most straightforward customization actions. So default highlight group is there only as a reasonably guessed guide.
- Highlighting cursorword on current line seems magnitude more frequent than highlighting cursorword also highlighted with diagnostic.
- Having high contrast for cursorword is not that good as it changes rapidly and will cause annoying flashing. Underline seems like a reasonable compromise between visibility, non-obtrusiveness, and terminal support.

**Reply by GnikDroy** [reply]
Sure. It does indeed not matter much. Just made the suggestion since it is what I observe to be the default behavior in other editors.
